Trellix/trellix-network-security/tests/riskware_callback.json

```
@@ -0,0 +1,63 @@
+        "dialect": "trellix-network-security",
+        "dialect_uuid": "bae128bb-98c6-45f7-9763-aad3451821e5"
+      }
+    },
+    "message": "CEF:0|Trellix|MPS|10.0.0.992057|RC|riskware-callback|1|rt=Sep 05 2023 16:46:47 UTC start=Sep 05 2023 16:46:47 UTC end=Sep 05 2023 16:46:47 UTC src=1.2.3.4 dst=5.6.7.8 request=hxxp://stan.mxp2142.com/abf858fda549bc190bd08eb75a07247bd98d194f57886d31b78b12ee01934bf147e3a36d2778243d1945d8a473515b6d196b33304340dfd578c64e47c8be025d7475f1090b8d3d34 cs1Label=sname cs1=Adware.Downloader.Generic act=notified dvc=3.4.5.6 dvchost=cms-nx5600-1.eng.fireeye.com dvcmac=e3:e9:d0:5e:ba:8e smac=6c:af:1a:fb:fe:a7 dmac=00:78:db:db:96:f6 spt=1072 dpt=80 cn1Label=vlan cn1=0 externalId=20966952 devicePayloadId=ae490699-29f0-4680-abb1-9db7ff757cad msg=risk ware detected:13436744 proto=tcp cs4Label=link cs4=https://cms-nx5600-1.eng.fireeye.com/detection/objects?uuid\\=ae490699-29f0-4680-abb1-9db7ff757cad cs6Label=channel cs6=GET /abf858fda549bc190bd08eb75a07247bd98d194f57886d31b78b12ee01934bf147e3a36d2778243d1945d8a473515b6d196b33304340dfd578c64e47c8be025d7475f1090b8d3d34 HTTP/1.1::~~Accept: */*::~~Proxy-Authorization: Basic ::~~User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36::~~Host: stan.mxp2142.com::~~Connection: Keep-Alive::~~::~~"
+  },
+  "expected": {
+    "message": "CEF:0|Trellix|MPS|10.0.0.992057|RC|riskware-callback|1|rt=Sep 05 2023 16:46:47 UTC start=Sep 05 2023 16:46:47 UTC end=Sep 05 2023 16:46:47 UTC src=1.2.3.4 dst=5.6.7.8 request=hxxp://stan.mxp2142.com/abf858fda549bc190bd08eb75a07247bd98d194f57886d31b78b12ee01934bf147e3a36d2778243d1945d8a473515b6d196b33304340dfd578c64e47c8be025d7475f1090b8d3d34 cs1Label=sname cs1=Adware.Downloader.Generic act=notified dvc=3.4.5.6 dvchost=cms-nx5600-1.eng.fireeye.com dvcmac=e3:e9:d0:5e:ba:8e smac=6c:af:1a:fb:fe:a7 dmac=00:78:db:db:96:f6 spt=1072 dpt=80 cn1Label=vlan cn1=0 externalId=20966952 devicePayloadId=ae490699-29f0-4680-abb1-9db7ff757cad msg=risk ware detected:13436744 proto=tcp cs4Label=link cs4=https://cms-nx5600-1.eng.fireeye.com/detection/objects?uuid\\=ae490699-29f0-4680-abb1-9db7ff757cad cs6Label=channel cs6=GET /abf858fda549bc190bd08eb75a07247bd98d194f57886d31b78b12ee01934bf147e3a36d2778243d1945d8a473515b6d196b33304340dfd578c64e47c8be025d7475f1090b8d3d34 HTTP/1.1::~~Accept: */*::~~Proxy-Authorization: Basic ::~~User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.153 Safari/537.36::~~Host: stan.mxp2142.com::~~Connection: Keep-Alive::~~::~~",
+    "event": {
+      "kind": "event",
+      "dataset": "riskware-callback",
+      "severity": 1,
+      "start": "2023-09-05T16:46:47Z",
+      "end": "2023-09-05T16:46:47Z",
+      "action": "notified",
+      "reason": "risk ware detected:13436744",
+      "url": "https://cms-nx5600-1.eng.fireeye.com/detection/objects?uuid\\=ae490699-29f0-4680-abb1-9db7ff757cad",
+      "category": ["network"],
```

Could you update the parser to categorize `riskware-callback` event as `intrusion_detection`?
```suggestion
      "category": ["intrusion_detection"],
```